proxy/src/config.rs

```diff
@@ -1,9 +1,56 @@
 //! Configuration vital to the setup and alteration of the application.
 
+use crate::error;
 use directories::ProjectDirs;
+use std::ffi;
+use std::path;
 
 /// Returns the directories to locate all application state.
 #[must_use]
 pub fn dirs() -> ProjectDirs {
     ProjectDirs::from("xyz", "radicle", "radicle-upstream").expect("couldn't build dirs")
 }
+
+/// Returns the path to a folder containing helper binaries.
+///
+/// # Errors
+///
+///   * Could not get the user home path from the HOME env variable
+pub fn bin_dir() -> Result<path::PathBuf, error::Error> {
+    let home_dir = std::env::var("HOME").expect("Couldn't determine home dir from env");
+
+    Ok(path::Path::new(&home_dir).join(".radicle/bin"))
```

Comment:
As we occupy a shared namespace with other crates this should be extracted and treated as a shared resource. Similar to how we use the `librad` to get a the location of the monorepo.

Reply:
You mean: `pub const RADICLE_BIN_HOME: &str = ".radicle/bin";`?

Reply:
More like `~/.radicle` is a shared directory across multiple crates. Flagging this here that this might be concern for the entire radicle-* ecosystem.